docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '2'
+services:
+  app:
+    build: .
+    ports:
+      - 3000:3000
+    command: bash -c " npm start"
```

Comment:
I would define `volumes` & `command: tail -f /dev/null` in `docker-compose.override.yml`.
`docker-compose`  reads by default both `docker-compose.yml` and `docker-compose.override.yml`.

So the dev workflow would be: `docker-compose up -d && docker -it exec <containerName> bash`
Whereas for production you would run `docker-compose -f docker-compose.override.yml up`